modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidgetEnhanced.php

```diff
@@ -0,0 +1,639 @@
+<?php
+
+namespace Drupal\tide_content_collection\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldWidget\StringTextareaWidget;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\tide_automated_listing\SearchApiIndexHelper;
```

Comment:
@joshua-salsadigital  why `tide_automated_listing`?

Reply:
Updated, referenced an used the correct SearchApiIndexHelper.